etcdserver/util.go

```diff
@@ -159,6 +160,9 @@ func warnOfExpensiveGenericRequest(lg *zap.Logger, now time.Time, reqStringer fm
 			}
 			plog.Warningf("%srequest %q with result %q took too long (%v) to execute", prefix, reqStringer.String(), result, d)
 		}
+		if trace != nil {
+			trace.Log(lg)
+		}
```

Comment:
How about we put tracing behind an experimental flag that we can use to tune the time threshold that must be exceeded before traces are logged?

Thoughts @jingyih ?

Reply:
I am also in favor of an adjustable tracing threshold. 100ms of the warning threshold might make the log output quite noisy in a busy cluster. I imagine we want to set the tracing threshold to be somewhere around 1s.

Reply:
I think we still need a default tracing threshold. I added one const `rangeTraceThreshold` in `apply.go`. I set it to 100 millisecond temporarily. In the meantime, possibly we can still add a flag to set the customized threshold for diagnostic purposes.